apisix/plugins/api-breaker.lua

```diff
@@ -35,6 +37,23 @@ local schema = {
             minimum = 200,
             maximum = 599,
         },
+        break_response_body = {
+            type = "string"
+        },
+        headers = {
+            type = "array",
+            items = {
+                type = "object",
+                properties = {
+                    key = {
+                        type = "string"
```

Comment:
Let's add "minLength = 1" to avoid empty string

Reply:
fixed with a new commit